index.html

```diff
@@ -2162,6 +2163,57 @@ <h5>Privacy Policy</h5>
     }],
     &#8230;
 }
+</pre>
+					</section>
+					
+					<section id="preview">
+						<h5>Preview</h5>
+
+						<p>A creator of a Web Publication my choose to make a preview of the publication available and
+							link it to the Web Publication. A link to a preview can be included in the manifest for
+							this purpose. The preview may be referenced externally or could be included as
+							a resource of the publication.</p>
+
+						<p>The <dfn data-lt="preview">preview</dfn> is identified using the
+								<code>preview</code> link relation&#160;[[!iana-link-relations]].</p>
+
+						<p>It is RECOMMENDED that the privacy policy be provided in a human-consumable format, such as
```

Comment:
I guess this sentence is a leftover from cut-and-paste:-)

Reply:
Yea. I had actually left it in on purpose, and expanded to audio.  But, you're right, bet removed... as it better be human consumable!  :-)

Reply:
Actually -- looks like Matt expunged when he was doing his other tuned.  So, good!